docs/data-sources/service_token.md

```diff
@@ -17,20 +17,25 @@ description: |-
 
 ### Required
 
-- `service_token_id` (Number) ID of the service token
+- `service_token_id` (Number) The ID of the service token
 
 ### Read-Only
 
-- `id` (String) The ID of this resource.
+- `id` (String) The ID of the service token
 - `name` (String) Service token name
-- `service_token_permissions` (Set of Object) Permissions set for the service token (see [below for nested schema](#nestedatt--service_token_permissions))
-- `uid` (String) The UID of the service token (part of the token secret)
+- `service_token_permissions` (Block Set) Permissions set for the service token (see [below for nested schema](#nestedblock--service_token_permissions))
+- `uid` (String) Service token UID (part of the token)
 
-<a id="nestedatt--service_token_permissions"></a>
+<a id="nestedblock--service_token_permissions"></a>
 ### Nested Schema for `service_token_permissions`
 
 Read-Only:
 
-- `all_projects` (Boolean)
-- `permission_set` (String)
-- `project_id` (Number)
+- `all_projects` (Boolean) Whether or not to apply this permission to all projects for this service token
+- `permission_set` (String) Set of permissions to apply
+- `project_id` (Number) Project ID to apply this permission to for this service token
+- `writable_environment_categories` (Set of String) What types of environments to apply Write permissions to.
```

Comment:
Can we update [the example file](https://github.com/dbt-labs/terraform-provider-dbtcloud/blob/main/examples/resources/dbtcloud_service_token/resource.tf) + regenerate docs so that we show an example of how people can set restrictions at the env level?

Reply:
Done!